### PR TITLE
show exact date and time when you hover over the package date

### PR DESCRIPTION
**Pull request by tugberkugurlu**
What the title says. Would be good to have this.

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -364,7 +364,7 @@
                             @packageVersion.DownloadCount
                         </td>
                         <td>
-                            @packageVersion.LastUpdated.ToNuGetLongDateString()
+                            <span title="@packageVersion.LastUpdated.ToUniversalTime()">@packageVersion.LastUpdated.ToNuGetLongDateString()</span>
```
Comment (tugberkugurlu) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: I am not sure about `ToUniversalTime` and I am also not sure whether this is in UTC.

Comment (maartenba) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: LastUpdated is indeed in UTC.
Can you change the ".ToUniversalTime()" to a proper DateTime format, e.g. "O" or "u" - https://msdn.microsoft.com/en-us/library/az4se3k1(v=vs.110).aspx

Comment (tugberkugurlu) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: done.

**Comment by maartenba** [comment]
@tugberkugurlu Can you reopen this PR against the dev branch? Looks good otherwise.
Thanks!

**Pull request closed without merging**